### ISSUE TITLE
Cancel upload/download of large files by button

### ISSUE THREAD

**Issue by DecaTec** [nextcloud/windows-universal#47]
The user should be able to cancel the upload or download of files (especially if these are very large).
This can be done by clicking the return key (in the app or on the device).

Maybe there should be a 'Cancel' button for these operations - at least for the upcoming Windows (desktop) app.

**Comment by SunboX** [comment]
Good point. It doesn't hurt to put another button on these pages. It should be inside the CommandBar (that should also be added).
